### PR TITLE
Format the message with cloudevent

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: doc/message.md [hunk 1]
```diff
@@ -328,110 +328,75 @@ Will list the supported resources later.
...
+      "eventNamespace": "local-policy-namespace",
+      "message": "Policy local-policy-namespace/policy-limitrange was propagated to cluster kind-hub1-cluster1/kind-hub1-cluster1",
+      "reason": "PolicyPropagation",
+      "count": 1,
+      "source": {
+        "component": "policy-propagator"
+      },
+      "createdAt": "2024-02-05T03:53:25Z",
+      "policyId": "13b2e003-2bdf-4c82-9bdf-f1aa7ccf608d",
+      "compliance": "NonCompliant"
```
Comment (clyang82) [inline doc/message.md:357]: Can we have `clusterId` to indicate which managed cluster the message is from?

File: doc/message.md [hunk 2]
```diff
@@ -328,110 +328,75 @@ Will list the supported resources later.
 ```
 ## Topic: event
 Currently, the following resource events are supported:
-- **Policy**: create, update, delete
```
Comment (clyang82) [inline doc/message.md:331]: I remembered we have `delete` message. am I correct?

Comment (yanmxa) [reply doc/message.md:331]: I haven't detected the delete event when delete the policy

File: doc/message.md (hunk 1 above)
Comment (yanmxa) [reply doc/message.md:357]: This is a local root policy, So we can't add the managed cluster to the payload.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**